src/views/check-your-answers.njk

```diff
@@ -4,11 +4,7 @@
 
 {% set checkYourAnswers = "check-your-answers" %}
 {% set currentUrl = props.currentUrl %}
-
-{% set fullName = props.data.limitedPartnership.data.partnership_name %}
-{% if props.data.limitedPartnership.data.name_ending %}
-  {% set fullName = fullName + " " + props.data.limitedPartnership.data.name_ending %}
-{% endif %}
+{% set fullName = (props.data.limitedPartnership.data.partnership_name | upper) + " " + (props.data.limitedPartnership.data.name_ending | upper) %}
```

Comment:
Simplified this code as a name ending has to be present at this point in the journey.